src/aequitas/plotting.py

```diff
@@ -548,13 +631,13 @@ def plot_fairness_group(self, fairness_table, group_metric, ax=None,
             # determinations
             cb_green = '#1b7837'
             cb_red = '#a50026'
-            measure = self._metric_parity_mapping[group_metric]
-            measure_colors = [cb_green if val == True else
-                              cb_red for val in attribute_data[measure]]
+            parity = self._metric_parity_mapping[group_metric]
+            parity_colors = [cb_green if val == True else
```

Comment:
Are the choices `(True, False)`?

I would think this would do:

    [cb_green if val else cb_red for val in ...]